pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -99,7 +100,7 @@ func (pl *VolumeBinding) EventsToRegister() []framework.ClusterEventWithHint {
 		{Event: framework.ClusterEvent{Resource: framework.StorageClass, ActionType: framework.Add | framework.Update}},
 		// We bind PVCs with PVs, so any changes may make the pods schedulable.
 		{Event: framework.ClusterEvent{Resource: framework.PersistentVolumeClaim, ActionType: framework.Add | framework.Update}},
-		{Event: framework.ClusterEvent{Resource: framework.PersistentVolume, ActionType: framework.Add | framework.Update}},
+		{Event: framework.ClusterEvent{Resource: framework.PersistentVolume, ActionType: framework.Add | framework.Update}, QueueingHintFn: pl.isSchedulableAfterPersistentVolumeChange},
```

Comment:
I don't know why this plugin registers PV events. IMO, The pv events can not directly influence a pod's schedulability. During the process of binding a PV to a PVC, the PV are always updated first, then the PVC is updated. So the later PVC event may make the Pod become schedulable. so registering PV events maybe not necessary but it may accelerate the pod scheduling process (it doesn't wait for the pvc to be bound and put the pod into backoffQ).

It was added by this PR: https://github.com/kubernetes/kubernetes/pull/100003

cc @jsafrane @sanposhiho @cofyc @Huang-Wei @saad-ali